### ISSUE TITLE
Trigger enforcer separately

### ISSUE THREAD

**Issue by Floppy012** [kordamp/enforcer-gradle-plugin#25]
Is there a way to trigger the enforcer separately like the maven `enforcer:enforce` goal?

I'm trying to use the enforcer in a separate CI job that just does that. Also the enforcer should not be active in the other jobs.

**Comment by aalmiray** [comment]
No, there's no such compatible task. Although you can set a System property `enforcer.enabled` to control whether all enforcer rules are triggered or not.
